noxfiles/docker_nox.py

```diff
@@ -78,6 +79,7 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
         "test": {"tag": lambda: IMAGE_LOCAL, "target": "prod"},
         "dev": {"tag": lambda: IMAGE_LOCAL, "target": "dev"},
         "admin_ui": {"tag": lambda: IMAGE_LOCAL_UI, "target": "frontend"},
+        "privacy_center": {"tag": lambda: IMAGE_LOCAL_PC, "target": "frontend"}
```

Comment:
it might make sense to have a separate stage for privacy center, since often we're running only the admin ui

Reply:
Just so I understand- you mean have a new `target` for privacy-center instead of using the `frontend` target?

Right now I'm using the same stage for both, because they both require an install at the root `clients` folder. Even if we only run the admin-ui, we'll still need to do the same steps anyway. See https://github.com/ethyca/fides/pull/3105/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R96

Reply:
Same comment as the other threads - a new ticket for Docker optimization makes sense, and I agree that having some app-specific build stages will work best.